app/models/cohort_columns/last_homeless_visit.rb

```diff
@@ -27,9 +27,16 @@ def value(cohort_client, user) # OK
       # e.g.: {:project_name=>\"APR - Transitional Housing\", :date=>Mon, 30 Sep 2019, :project_id=>10}
       return unless lhv.present?
 
+      enforce_visibility = cohort_client.cohort.enforce_project_visibility_on_cells?
       lhv = JSON.parse(lhv)
       lhv.select do |row|
-        row['project_id'].in? user.visible_project_ids_enrollment_context
+        next row['project_id'].in?(user.visible_project_ids_enrollment_context) if enforce_visibility
+        next true unless cohort.only_window?
```

Comment:
Does this need to be:
```suggestion
        next true unless cohort_client.cohort.only_window?
```

Reply:
Pretty sure it doesn't, the base class has a cohort attribute that should be set.

Reply:
This is done here: https://github.com/greenriver/hmis-warehouse/blob/ede56cc0dd04b76a069d9d8d1e6f27b0dfa19b23/app/controllers/cohorts/clients_controller.rb#L69